docs/source/processors/scratch.rst

```diff
@@ -25,6 +31,11 @@ You can test the output of your Scratch block text at
     (for example: Django) to create the images.
     See :ref:`accessing-scratch-image-data` section below.
 
+.. note::
+
+    The following examples assume usage of the fenced code extension, by having
```

Comment:
Yes all the examples we show have to uses the fenced code extension. It does work without the extension though see the test: example_standard_markdown_block.